### PR TITLE
README.md - Fix link to calico-containers repository

### PR DESCRIPTION
**Pull request by bjwbell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,6 @@ To build the Calico Networking Plugin for CNI locally, clone this repository and
 
 [cni]: https://github.com/appc/cni
 [config]: configuration.md
-[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md
+[calico-containers]: https://github.com/projectcalico/calico-containers/blob/master/README.md
```
Comment (caseydavenport) [inline README.md]: I think we actually want to link to docs.projectcalico.org instead - we've moved all the docs and guides over there, so calico-containers doesn't have it anymore.

**Comment by caseydavenport** [comment]
Thanks for the PR - one minor comment (we've actually moved the docs around a bit, so this should be updated to point to docs.projectcalico.org instead!)

File: README.md (hunk 1 above)
Comment (gunjan5) [inline README.md]: @caseydavenport would it be http://docs.projectcalico.org/master/reference/advanced/calico-cni or http://docs.projectcalico.org/master/getting-started/kubernetes/ ?

**Comment by bjwbell** [comment]
@caseydavenport I can squash the commits.

**Comment by caseydavenport** [comment]
@bjwbell please do squash and then LGTM!

**Pull request merged**